recipes/libsecret/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
```

Comment:
```suggestion

required_conan_version = ">=1.32.0"
```

`validate()` is 1.32